### PR TITLE
Feat/enhancement2

### LINKED ISSUE #35: Allow to provide account credentials via environment variables
Let ionosctl source its credentials for account authentication from environment variables as an alternative to the config file based approach that is already implemented.

This would mean that you would not have to store the credentials, potentially unencrypted in a config file but in a transient environment variables.

This use of environment variables then could be combined with a tool like [envdir](https://github.com/direnv/direnv) that can load environment variables on the dynamically on the fly when you enter a project folder.

In combination with a tool like [gopass](https://github.com/gopasspw/gopass) those credentials could even be stored encrypted and then only dynamically loaded, encrypted and store in a transient environment variable by envdir.

The naming of the environment variables should be aligned with environment variables that we might are already use in other IONOS Cloud SDKs or tools like Terraform/Ansible.

We also need to define an order of precedence for different ways/locations that ionosctl is looking for credentials - falling back from one to the next.

### PR DESCRIPTION
**Pull request by gabisavu98**
fixes #35

### CODE REVIEW

File: commands/login_test.go
Comment (AnaMariaMoga) [inline commands/login_test.go:10]: In Go, it is recommended to separate remote imports like `ionoscloud "github.com/ionos-cloud/sdk-go/v5"` for the other ones.

File: pkg/config/config.go
Comment (AnaMariaMoga) [inline pkg/config/config.go:10]: Some comment as above.

File: pkg/config/config.go
Comment (AnaMariaMoga) [inline pkg/config/config.go:41]: Clean-up :D

**Review by AnaMariaMoga — APPROVED** [review]
Good job overall, just small comments. I approve this, and after fixing the Windows tests - you can merge it.

File: commands/login_test.go
Comment (gabisavu98) [reply commands/login_test.go:10]: done

File: pkg/config/config.go
Comment (gabisavu98) [reply pkg/config/config.go]: done

File: pkg/config/config.go
Comment (gabisavu98) [reply pkg/config/config.go]: done

File: README.md
Comment (AnaMariaMoga) [inline README.md:90]: After today;s demo, I think it is important to add in here that environment variables have priority to config file. This applies also to the other README.md and to v6.

Comment (gabisavu98) [reply README.md]: changed

File: pkg/config/config.go
Comment (AnaMariaMoga) [inline pkg/config/config.go:63]: Please remove the fmt.Printf in here and below, eventually, you can add it to the `errors.New("no permission for the config file, expected 600, got "+permNumber)` to be more explicit, but remaining this way will affect the json/text format printing of the tool.

Comment (gabisavu98) [reply pkg/config/config.go]: done

**Pull request merged**
